src/js/utils/ServiceSpecUtil.js

```diff
@@ -0,0 +1,33 @@
+import PodSpec from '../structs/PodSpec';
+import Util from './Util';
+
+var ServiceSpecUtil = {
+
+  /**
+   * Creates a new PodSpec with the scaling specifications
+   * changed to fixed, and with the number of instances defined
+   * from the second argument.
+   *
+   * @param {PodSpec} spec - The spec to change
+   * @param {number} instancesCount - The new number of instances
+   * @returns {PodSpec} - Returns a new PodSpec instance
+   */
+  setFixedScaling(spec, instancesCount) {
```

Comment:
I thought you meant introducing a new `scaleXXX` action on `MarathonStore`. Having a single utility for both specs sounds better.